config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -679,6 +679,48 @@ presubmits:
       testgrid-tab-name: release-secrets-store-csi-driver-e2e-azure
       description: "Run e2e test with azure provider for Secrets Store CSI driver release."
       testgrid-num-columns-recent: '30'
+  - name: release-secrets-store-csi-driver-e2e-gcp
```

Comment:
This job will only be triggered during release and isn't possible to test otherwise. Instead use `pull-secrets-store-csi-driver-e2e-gcp` that can be optionally run for testing. Replace your changes with this

```yaml
  - name: pull-secrets-store-csi-driver-e2e-gcp
    cluster: k8s-infra-prow-build
    decorate: true
    decoration_config:
      timeout: 25m
    always_run: false
    optional: true
    path_alias: sigs.k8s.io/secrets-store-csi-driver
    branches:
    - ^main$
    - ^release-*
    labels:
      # this is required because we want to run kind in docker
      preset-dind-enabled: "true"
      # this is required to make CNI installation to succeed for kind
      preset-kind-volume-mounts: "true"
    spec:
      serviceAccountName: secrets-store-csi-driver-gcp
      containers:
      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240803-cf1183f2db-master
        command:
          - runner.sh
        args:
          - bash
          - -c
          - >-
            make e2e-bootstrap e2e-helm-deploy e2e-gcp
        securityContext:
          privileged: true
        resources:
          requests:
            cpu: "4"
            memory: "4Gi"
    annotations:
      testgrid-dashboards: sig-auth-secrets-store-csi-driver, sig-auth-secrets-store-csi-driver-presubmit
      testgrid-tab-name: pr-secrets-store-csi-driver-e2e-gcp
      description: "Run e2e test with gcp provider for Secrets Store CSI driver."
      testgrid-num-columns-recent: '30'
```

Reply:
Done